list-sensors.cpp

```diff
@@ -263,7 +263,7 @@ void printSensorData(const std::string& service, const std::string& path)
     size_t folder_pos = path.rfind('/', name_pos - 1);
 
     // row format string
-    constexpr auto row_fmt = "%-18s %8s %7s %-4s %7s %7s %7s %7s %7s\n";
+    constexpr auto row_fmt = "%-19s %8s %7s %-3s %7s %7s %7s %7s %7s\n";
```

Comment:
`Unit` contains 4 characters. As a result, it pushes all other columns headers and looks weird.
```
=== current ===
Name                  Status   Value Unit      LC     LNC     UNC      UC      NR
PSU1_Input_Current        OK   0.367 A       N/A     N/A     N/A     N/A     N/A
PSU1_Output_Current       OK   4.117 A       N/A     N/A 105.000 125.000     N/A
```

Reply:
Good catch. I missed that.

Reply:
I was wondering why it was 4 and didn't notice that it's the length of the column name